mapper/helper.go

```diff
@@ -11,3 +25,74 @@ func EqualFoldContains(arr []string, str string) bool {
 	}
 	return false
 }
+
+// GetHRP fetches hrp for address formatting.
+func GetHRP(networkIdentifier *types.NetworkIdentifier) (string, error) {
+	var hrp string
+	switch networkIdentifier.Network {
+	case FujiNetwork:
+		hrp = constants.GetHRP(constants.FujiID)
+	case MainnetNetwork:
+		hrp = constants.GetHRP(constants.MainnetID)
+	default:
+		return "", errUnrecognizedNetwork
+	}
+
+	return hrp, nil
+}
+
+// UnmarshalJSONMap converts map[string]interface{} into a interface{}.
+func UnmarshalJSONMap(m map[string]interface{}, i interface{}) error {
```

Comment:
I think there's very likely a better way to transform the information in `m` into an `i`. This is pretty error prone due to matching on exact spelling/casing of keys in the map / JSON tags on `i`.

Reply:
We simply moved the existing private function from service/helper.go to here as-is for reuse in the new code. We can look at refactoring the mechanism in a future PR.